Networking/WebSockets/BuiltInWebSocket.cc

```diff
@@ -15,6 +15,7 @@
 #include "HTTPLogic.hh"
 #include "Certificate.hh"
 #include "CookieStore.hh"
+#include "c4Certificate.hh"
```

Comment:
I don't think this is necessary since you're just passing a pointer around.

Reply:
done